test/test_bmp.cpp

```diff
@@ -0,0 +1,319 @@
+#include <catch2/catch_all.hpp>
+#include "../src/bmp.hpp"
+#include "../src/pixel.hpp"
+
+#include <bit>
+#include <iostream>
+
+using Catch::Matchers::ContainsSubstring;
+
+TEST_CASE("Read BMP files" , "[bmp][read]") {
+    std::string bmp_path;
```

Comment:
If all the `SECTION`s happen to independently each need a var `bmp_path`, I'd not extract that out of those `SECTION`s.  (Otherwise, you might wonder whether in some cases the value assigned to it in one `SECTION` could be relevant still after leaving that `SECITON`.)